cmd/latest_sdk_version.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+func latestSdkVersionCmd() *cli.Command {
+	var config LatestSdkVersionConfig
+	return &cli.Command{
+		Name:  "latest-sdk-version",
+		Usage: "get the latest SDK version",
+		Flags: config.flags(),
+		Action: func(ctx *cli.Context) error {
+			return getLatestSdkVersion(config)
+		},
+	}
+}
+
+type LatestSdkVersionConfig struct {
+	Lang string
+}
+
+func (p *LatestSdkVersionConfig) flags() []cli.Flag {
+	return []cli.Flag{
+		langFlag(&p.Lang),
+	}
+}
+
+type Version struct {
```

Comment:
```suggestion
type GitHubApiVersion struct {
```

Just to avoid confusion with the general term for readers, or you can just change the code below to use an anonymous struct like `version := struct { Name string `json:"name"` }{}`.